### ISSUE TITLE
Searching UNI and ANA

### ISSUE THREAD

**Issue by faulhaber** [PhiloBiblon/philobiblon-ui#124]
To test the problem of searching UNI and ANA, see:
[BETA texid 1029 (Alfonso X, Siete partidas)](https://philobiblon.upf.edu/xtf/servlet/org.cdlib.xtf.dynaXML.DynaXML?source=unified/Display/1029BETA.Work.xml&style=Work.xsl%0A%0A%20%0A%20%0A%20&gobk=http%3A%2F%2Fphilobiblon.upf.edu%2Fxtf%2Fservlet%2Forg.cdlib.xtf.crossQuery.CrossQuery%3Frmode%3Dphilobeta%26everyone%3Dtexid+1029%26creator%3D%26title%3D%26incipit%3D%26explicit%3D%26assocname%3D%26daterange%3D%26placeofcomposition%3D%26subject%3D%26text-join%3Dand%26browseout%3Dwork%26sort%3Dmoniker)
[BETA texid 1029 (pb.cog)](https://philobiblon.cog.berkeley.edu/wiki/Item:Q43297) linked to
[BETA cnum 130 (pb.cog)](https://philobiblon.cog.berkeley.edu/wiki/Item:Q160)

Update of [issue#89](https://github.com/PhiloBiblon/philobiblon-ui/issues/89)

**Comment by jmformenti** [comment]
SPARQL query to get all cnum related to a given texid (in this example [Q43297](https://philobiblon.cog.berkeley.edu/wiki/Item:Q43297)):
```
PREFIX wd:<https://philobiblon.cog.berkeley.edu/entity/>
PREFIX wdt:<https://philobiblon.cog.berkeley.edu/prop/direct/>

SELECT ?item_cnum {
  ?item_cnum wdt:P476 ?item_cnum_pbid .
  FILTER regex(?item_cnum_pbid, '(.*) cnum ') .
  ?item_cnum wdt:P590 wd:Q43297 .
}
```

**Comment by faulhaber** [comment]
I will test this today

**Comment by jmformenti** [comment]
Already implemented in #142 and #73
